client/signup/steps/intent/intent-screen.tsx

```diff
@@ -33,6 +33,14 @@ const useIntents = ( { translate }: Pick< Props, 'translate' > ): Intent[] => {
 			value: 'build',
 			actionText: translate( 'Start building' ),
 		},
+		{
+			key: 'sell',
+			title: translate( 'Sell' ),
+			description: translate( 'Sell your products' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Sell your products online' )` *ES Score: 7*